zk-token-sdk/src/lib.rs

```diff
@@ -1,3 +1,9 @@
+// Deprecate the crate
+#![deprecated(
+    note = "use the `zk-sdk` instead: https://github.com/solana-program/zk-elgamal-proof/tree/main/zk-sdk"
+)]
```

Comment:
Also, can you add a deprecated version? Are you planning to backport this to v2.3 and removing in master?

Reply:
Okay, yeah I'll try backporting this to v2.3, so I specified the version to be 2.3.8, which is the next v2.3 version. After a successful backport, I'll remove the crate from master.